tests/runner.py

```diff
@@ -549,8 +549,10 @@ def verify_es5(self, filename):
     es_check = shared.get_npm_cmd('es-check')
     # use --quiet once its available
     # See: https://github.com/dollarshaveclub/es-check/pull/126/
+    es_check_env = os.environ.copy()
+    es_check_env['PATH'] = os.path.dirname(config.NODE_JS[0] if type(config.NODE_JS) is list else config.NODE_JS) + os.pathsep + es_check_env['PATH']
```

Comment:
```suggestion
    es_check_env['PATH'] = os.path.dirname(config.NODE_JS[0] + os.pathsep + es_check_env['PATH']
```
It should already be a list, `config.py` runs `listify` internally.